calico_containers/pycalico/datastore.py

```diff
@@ -56,9 +65,12 @@
 cali123456789ab.
 """
 
-# The default node AS number
+# The default node AS number.  Note that this value is also hard-coded into
```

Comment:
So it doesn't need to be int he bird template right? Why did you add a comment to say it does? I think I must be missing something.